docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md

```diff
@@ -115,13 +117,12 @@ information, refer to [Enable Local Harbor Registry](./local-registry.md).
    use the Palette Edge CLI to upload all images in a cluster profile to an external registry.
 
 9. In the Kubernetes layer of your cluster profile, remove `AlwaysPullImages` from
-   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins` and add it to
    `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins`.
 
    The resulting layer configuration should look like the following.
 
    ```yaml
-   disable-admission-plugins: AlwaysPullImages,AlwaysAdmit
+   disable-admission-plugins: AlwaysAdmit
```

Comment:
@lennessyy May be we can skip any change in `disable-admission-plugins` section. It's not there in released docs https://docs.spectrocloud.com/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry/ and for above external registry's changes, we don't need to make any change in `disable-admission-plugins`.